docs/book/commands-chain.md

```diff
@@ -10,75 +10,58 @@ namespace MyApp\Command;
 
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputOption;use Symfony\Component\Console\Output\OutputInterface;
 
-class FirstCommand extends Command 
+class FirstCommand extends Command
 {
+    /** @var string */
     protected static $defaultName = 'first-command';
 
     protected function configure() : void
     {
         $this->setName(self::$defaultName);
+        $this->addOption('name', null, InputOption::VALUE_REQUIRED, 'Module name');
     }
-    
+
     protected function execute(InputInterface $input, OutputInterface $output) : int
     {
-        $output->writeln('First command.');
- 
+        $output->writeln('First command: ' . $input->getOption('name'));
+
         return 0;
     }
 }
-``` 
+```
 
 `MyApp\Command\SecondCommand.php`:
 
 ```php
 namespace MyApp\Command;
 
-use Laminas\Cli\CommandListenerInterface;
 use Symfony\Component\Console\Command\Command;
-use Symfony\Component\Console\Event\ConsoleEvent;
-use Symfony\Component\Console\Helper\QuestionHelper;
 use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Input\InputOption;use Symfony\Component\Console\Output\OutputInterface;
 
-class FirstCommand extends Command implements CommandListenerInterface
+class FirstCommand extends Command
```

Comment:
The name of the class should be `SecondCommand `.

Reply:
Nice catch, thanks, fixed now!